lib/trino-orc/src/main/java/io/trino/orc/StripeReader.java

```diff
@@ -439,13 +440,13 @@ private Set<Integer> selectRowGroups(StripeInformation stripe, Map<StreamId, Lis
     {
         int rowsInRowGroup = this.rowsInRowGroup.orElseThrow(() -> new IllegalStateException("Cannot create row groups if row group info is missing"));
 
-        int rowsInStripe = stripe.getNumberOfRows();
+        long rowsInStripe = stripe.getNumberOfRows();
         int groupsInStripe = ceil(rowsInStripe, rowsInRowGroup);
```

Comment:
I've pushed changes to keep this as int since the below iteration over rowGroup expects int